sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Bindings/ServiceBusAttributeBindingProvider.cs

```diff
@@ -33,6 +33,7 @@ internal class ServiceBusAttributeBindingProvider : IBindingProvider
                 new MessageArgumentBindingProvider(),
                 new StringArgumentBindingProvider(),
                 new ByteArrayArgumentBindingProvider(),
+                new BinaryDataArgumentBindingProvider(),
```

Comment:
Wasn't something added to WebJobs SDK to support binary data everywhere? Wonder why it doesn't light up for us.

Reply:
Possibly, but wouldn't that just work when dealing with strings/bytes? Any time you have a rich model type (e.g. ServiceBusMessage), you need to specify how to convert it to a string or BinaryData or bytes.